setup.cfg

```diff
@@ -26,7 +26,7 @@ ignore = E501
 exclude = setup_package.py,conftest.py,__init__.py
 
 [metadata]
-package_name = astrowidgets
+name = astrowidgets
```

Comment:
Looks like this is causing https://travis-ci.org/astropy/astrowidgets/jobs/551127071#L1442

Reply:
Ops. Fixed!